src/commands/mobile/scrollDownToElement.js

```diff
@@ -0,0 +1,67 @@
+import util from "util";
+import BaseCommand from "../../base-mobile-command";
+
+const MAX_ATTEMPTS = 10;
+
+const ScrollDownToElement = function (nightwatch = null) {
+  BaseCommand.call(this, nightwatch);
+  this.cmd = "scrolldowntoelement";
+};
+
+util.inherits(ScrollDownToElement, BaseCommand);
+
+ScrollDownToElement.prototype.command = function (xStart, yStart, elementLocateStrategy, elementSelector, cb) {
+  const self = this;
+  this.selector = elementSelector;
+  this.using = elementLocateStrategy;
+  this.cb = cb;
+
+  this.failureMessage = `Selector '${this.using}:${this.selector}' `
+    + `was not visible after scrolling ${MAX_ATTEMPTS} times`;
+
+  const scrollToFindElement = function (attempts) {
+
+    if (attempts < MAX_ATTEMPTS) {
+      self.client.api.getMobileElConditional(elementLocateStrategy,
+        elementSelector, 1000,
+          (result) => {
+            if (!result) {
+              if (self.client.api.capabilities.platformName === 'Android') {
```

Comment:
better to turn `self.client.api.capabilities.platformName` to lower case and check with `android`